fem/bilininteg_vectorfe.cpp

```diff
@@ -184,6 +185,18 @@ void PAHdivMassApply3D(const int D1D,
                        const Vector &x_,
                        Vector &y_);
 
+void PAHdivMassApply(const int dim,
```

Comment:
In this file, you can now remove the declarations of `PAHdivMassApply2D` and `PAHdivMassApply3D`.

Reply:
Good point, fixed.